recipes/jupyterlite-core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jupyterlite-core" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlite_core-{{ version }}.tar.gz
+  sha256: fea7ffe880f6efb16cd39855dd459ec3cf87b238de076dae518033926af1150a
+
+build:
+  skip: true  # [py<38]
```

Comment:
doesn't need skip

Reply:
right I think it's added by `grayskull` automatically